components/cqrs/command_bus.go

```diff
@@ -3,18 +3,21 @@ package cqrs
 import (
 	"context"
 
+	"github.com/ThreeDotsLabs/watermill"
 	"github.com/pkg/errors"
 
 	"github.com/ThreeDotsLabs/watermill/message"
 )
 
 // CommandBus transports commands to command handlers.
 type CommandBus struct {
-	publisher     message.Publisher
-	generateTopic func(commandName string) string
-	marshaler     CommandEventMarshaler
+	publisher message.Publisher
+
+	config CommandProcessorConfig
```

Comment:
the struct is named `CommandBus`, but the config is named `CommandProcessorConfig`
is taht ok?

Reply:
yeah, it was a leftover 😅 but it was already re-written ten times here: https://github.com/ThreeDotsLabs/watermill/pull/367 (and should be fine now)